velox/dwio/parquet/reader/ParquetReaderUtil.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/dwio/parquet/reader/ParquetReaderUtil.h"
+
+namespace facebook::velox::parquet {
+
+common::CompressionKind thriftCodecToCompressionKind(
+    thrift::CompressionCodec::type codec) {
```

Comment:
You could make the codec a const reference.

Reply:
Since codec is an enum (int), the const reference is unnecessary.

Reply:
Yes, right now it is just a const and not a const reference. So just remove const again please.